.github/workflows/mango_base_publish_to_pypi.yml

```diff
@@ -1,16 +1,16 @@
-name: Publish to Pypi
+name: Publish mango to Pypi
 
 on:
   push:
     tags:
-      - "v*"
+      - 'mango_base_*'
```

Comment:
Why not mango*?

Reply:
Check also the .yml names (mango_base_...yml)

Reply:
Because if then we use mango_time_series for the tags of the other build it would trigger this one as well.

The other way is to have the tags be different, so mango for the main, ts_ for the time series, that way it may be easier.

I will change the name of the yaml files.